rte/mo_rte_lw.F90

```diff
@@ -132,7 +133,7 @@ function rte_lw(optical_props, top_at_1, &
     !
     if(any([size(sfc_emis,1), size(sfc_emis,2)] /= [nband, ncol])) &
       error_msg = "rte_lw: sfc_emis inconsistently sized"
-    if(any(sfc_emis < 0._wp .or. sfc_emis > 1._wp)) &
+    if(any_vals_outside(sfc_emis, 0._wp, 1._wp)) &
```

Comment:
fancy

Reply:
Needed this to get the checks to run on the GPU, didn't I? I'd rather not complicate things if not.